index.html

```diff
@@ -149,6 +149,13 @@
       </a>
       <a class="action-button" href="/groups/index.html"> Discord Groups </a>
       <a class="action-button" href="/standup/index.html"> Standup Updates</a>
+      <a
+        id="application-button"
+        href="/applications"
```

Comment:
Hi @vinayak-trivedi can we have a better naming here, I would not have been able to understand what application does, if this was viewed without any context by me.

Reply:
Discussed this, it takes to Applications page.